### PR TITLE
Update Bobcat Diagnostic Screen 

### PR DESCRIPTION
**Pull request by JackeyTY**

### CODE REVIEW

**Comment by JackeyTY** [comment]
Hi can you help us check the Japanese and Korean translation? We don't have relevant personnel and I used google translation on the English text. Thank you so much!

File: src/makers/bobcat/hotspots.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ const BOBCAT_MINER_300 = {
   translations: {
     en: {
       diagnostic:
-        '<b><white>Diagnostic support allows Bobcat to identify issues with your Hotspot in a secure way.</white></b>\n\nBobcat will never have access to private keys and will only ever be able to access your Hotspot and not any other devices on your Network.\n\nIf you would like to opt-out of diagnostic support please email <purple><b>support@bobcatminer.com</b></purple> from the email used to purchase the Hotspot.',
+        '<b><white>Diagnostic support allows Bobcat to identify issues with your Hotspot in a secure way.</white></b>\n\nBobcat will never have access to private keys and will only ever be able to access your Hotspot and not any other devices on your Network.\n\nPlease check out our troubleshooting guide at <purple><b>https://www.bobcatminer.com/faq</b></purple> or use the local diagnoser to see hotspot real time operational data.',
```
Comment (cokes518) [inline src/makers/bobcat/hotspots.ts:11]:
```suggestion
        '<b><white>Diagnostic support allows Bobcat to identify issues with your Hotspot in a secure way.</white></b>\n\nBobcat will never have access to private keys and will only ever be able to access your Hotspot and not any other devices on your Network.\n\nPlease check the troubleshooting guide at <purple><b>https://www.bobcatminer.com/faq</b></purple> or use the local diagnoser to see Hotspot real-time operational data.',
```

**Comment by JackeyTY** [comment]
Thanks Coco!

**Comment by cokes518** [comment]
all good from me @tyler-whitman over to you

**Review by tyler-whitman — APPROVED** [review]

**Pull request merged**